components/dialog/visibility.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Visibility
+page_title: Dialog Visibility
+description: Visibility of the Dialog for Blazor.
+slug: dialog-visibility
+tags: telerik,blazor,dialog,visibility
+published: True
+position: 7
+---
+
+# Dialog Visibility
```

Comment:
I am not sure about the need for a separate article about Visibility. All the other examples already demonstrate this.